### PR TITLE
Add reset database endpoints

### PR DESCRIPTION
**Pull request by Mubangizi**
#### What this PR does
- Adds reset database endpoints
- Can be tested on `/projects/{project_id}/databases/{database_id}/reset` for users
- And `/databases/{database_id}/reset`

### CODE REVIEW

File: app/helpers/database_service.py [hunk 1]
```diff
@@ -117,8 +130,31 @@ def delete_database(self, db_name):
                 cursor.close()
                 connection.close()
 
-    # Show all databases
+    def reset_database(self, db_name=None, user=None, password=None):
+        try:
+            connection = self.create_connection()
+            user_rights = self.check_user_db_rights(
+                db_name=db_name, user=user, password=password)
+            print(user_rights)
```
Comment (MutegekiHenry) [inline app/helpers/database_service.py:138]: clean out the print

**Review by MutegekiHenry — COMMENTED** [review]
@Mubangizi good work! Just small cleaning up to do.

File: app/helpers/database_service.py (hunk 1 above)
Comment (Mubangizi) [reply app/helpers/database_service.py]: I have fixed this

**Review by MutegekiHenry — APPROVED** [review]
LGMT

**Pull request merged**